core/src/main/java/jenkins/diagnostics/RootUrlNotSetMonitor.java

```diff
@@ -52,15 +52,13 @@ public String getDisplayName() {
     @Override
     public boolean isActivated() {
         JenkinsLocationConfiguration loc = JenkinsLocationConfiguration.get();
-        return loc != null && (
-                loc.getUrl() == null || !UrlHelper.isValidRootUrl(loc.getUrl())
-        );
+        return loc.getUrl() == null || !UrlHelper.isValidRootUrl(loc.getUrl());
     }
     
     // used by jelly to determined if it's a null url or invalid one
     @Restricted(NoExternalUse.class)
     public boolean isUrlNull(){
         JenkinsLocationConfiguration loc = JenkinsLocationConfiguration.get();
-        return loc != null && loc.getUrl() == null;
+        return loc.getUrl() == null;
```

Comment:
If you don't need the null check on `loc` here than can you remove it from `isActivated` as well? Looks like it is `@CheckForNull` so I think you want to add it back.

Reply:
It's already the case for `isActivated` and the `JenkinsLocationConfiguration.get()` is `@Nonnull`.

Do you have the lastest version or perhaps it's me

Reply:
I was looking at an old version, this looks good to me. Sorry for the noise!